packages/app-root/src/app/layout.js

```diff
@@ -0,0 +1,19 @@
+import RootLayout from '@/components/RootLayout'
+import StyledComponentsRegistry from './style-registry'
+
+export const metadata = {
+  title: 'Zooniverse',
+  description: 'People-powered Research'
+}
+
+export default function NextLayout({ children }) {
+  return (
+    <html lang='en'>
+      <body style={{ margin: 0 }}>
```

Comment:
Avoid inline styles. The document body should be styled with CSS, eg. (since we're using styled components)
```js
const StyledBody = styled.body`
  margin: 0;
`
```

Reply:
Good point - I'll change this margin style to the above function